lib/mode-loggedin/index.js

```diff
@@ -146,6 +147,7 @@ LoggedinPanel.prototype.bindAll = function() {
   this.query('.a0-strategy .a0-zocial[data-strategy]').a0_on('click', function (e) {
     stop(e);
 
+    widget.emit('loggedin submit', widget.options);
```

Comment:
pass along lastUsedConnection and getLoggedInAuthParams... something like when triggered _signin